src/components/header/Header.tsx

```diff
@@ -6,19 +8,40 @@ import TokenService from '../../services/TokenService'
 
 import './Header.css'
 
-const Header = () => {
+interface HeaderProps {
+  isLogged: boolean
+  setIsLogged: React.Dispatch<React.SetStateAction<boolean>>
+}
+
+const Header = ( { isLogged, setIsLogged } : HeaderProps) => {
   const navigateForLogout = useNavigate()
+  const navigeteForLogin = useNavigate()
 
 
   const handleLogout = () => {
     TokenService.deleteLocalStorage()
-    navigateForLogout('/')
+    navigateForLogout('/mainpage')
+    setIsLogged(false)
+  }
+
+  const handleLogin = () => {
+    navigeteForLogin('/login')
+    setIsLogged(true)
   }
 
   return (
-    <div className='header_container'>
-      <button className='logout_button' onClick={handleLogout}>Logout <LogoutIcon/></button>
-    </div>
+    <>
+    {isLogged
+     ?
+      <div className='header_container'>
+        <button className='logout_button' onClick={handleLogout}>Logout <LogoutIcon/></button>
+      </div>
+    :
+      <div className='header_container'>
+        <button className='login_button' onClick={handleLogin}>Login <LogoutIcon/></button>
```

Comment:
Should this be login icon or logout as it is now?

Reply:
This is conditional rendering, whitch inspect if user logged. If user logged, header will show logout Icon because user can just logout.